TFNetworkLayer.py

```diff
@@ -485,6 +492,16 @@ def get_sub_layer(self, layer_name):
     """
     return None
 
+  @classmethod
+  def get_sub_layer_out_data_from_opts(cls, layer_name, parent_layer_kwargs):
+    """
+    Called by _TemplateLayer.get_sub_layer(). Gets a Data template for the sub-layer with name 'name'. Also returns
+    the network the sub-layer is in and the class type of the sub-layer. There is no good default behaviour here,
+    as this heavily depends on how the current layer uses sub-layers.
+    :rtype: (Data, TFNetwork, type)|None
```

Comment:
Documentation (`:return:`, layer_name, parent_layer_kwargs) missing.

Reply:
`layer_name` type is not specified...

Reply:
Oops, fixed.